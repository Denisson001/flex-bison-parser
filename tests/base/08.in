s0jadk-342jhdf-3qfioz;xdjf-asa-dfuj3-4