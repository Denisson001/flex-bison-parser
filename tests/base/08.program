{
    read(line_s);
    flag = 1;
    while (flag == 1) {
        k = 0;
        while (k + 1 < len(line_s)) {
            if (line_s[k] > line_s[k + 1]) {
                tmp = line_s[k];
                line_s[k] = line_s
            }
        }
    }
}