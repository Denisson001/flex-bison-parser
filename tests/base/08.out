-----0233344;aaaddddffffhijjjjkoqssuxz
